mandatory/2-function-creation.js

```diff
@@ -25,7 +40,14 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  for(i = 0; i < arrayOfStrings.length; i++){
+    arrayOfStrings[i] = arrayOfStrings[i].replace("/", "");
+    arrayOfStrings[i] = arrayOfStrings[i].trim();
+    arrayOfStrings[i] = arrayOfStrings[i].toLowerCase();
+  }
+ return arrayOfStrings;
+}
```

Comment:
This is fine. You could also chain the 3 functions into 1 line of code, but this is personal preference - either way is correct.

Reply:
I've changed it to one line of code.